vector-im/element-android
# Disable live location sharing feature by default in debug config

## Type of change


## Content


Removing BuildConfigField for live location sharing activation. Disable the feature by default.

## Motivation and context


Closes #5751 

## Screenshots / GIFs





## Tests



- Run the app and check the live location sharing feature is disabled by default (no option for live location sharing)
- Enable the feature using the corresponding feature flag
- Check the live location sharing option is displayed

## Tested devices

- OS version(s): Android 11

## Checklist
